uv.lock

```diff
@@ -784,9 +844,18 @@ wheels = [
     { url = "https://files.pythonhosted.org/packages/00/2e/d53fa4befbf2cfa713304affc7ca780ce4fc1fd8710527771b58311a3229/click-8.1.7-py3-none-any.whl", hash = "sha256:ae74fb96c20a0277a1d615f1e4d73c8414f5a98db8b799a7931d1582f3390c28", size = 97941 },
 ]
 
+[[package]]
+name = "cloudpickle"
+version = "2.2.1"
+source = { registry = "https://pypi.org/simple" }
+sdist = { url = "https://files.pythonhosted.org/packages/5f/51/913ecca3970a2227cf4d5e8937df52cc28f465ac442216110b8e3323262d/cloudpickle-2.2.1.tar.gz", hash = "sha256:d89684b8de9e34a2a43b3460fbca07d09d6e25ce858df4d5a44240403b6178f5", size = 60800 }
+wheels = [
+    { url = "https://files.pythonhosted.org/packages/15/80/44286939ca215e88fa827b2aeb6fa3fd2b4a7af322485c7170d6f9fd96e0/cloudpickle-2.2.1-py3-none-any.whl", hash = "sha256:61f594d1f4c295fa5cd9014ceb3a1fc4a70b0de1164b94fbc2d854ccba056f9f", size = 25944 },
+]
+
 [[package]]
 name = "cohere"
-version = "5.5.8"
+version = "5.11.0"
 source = { registry = "https://pypi.org/simple" }
```

Comment:
OK, I understand the reason of the CI error.
Recently, `cohere` sdk deleted the few class from definition.
But, mirascope uses the classes.

https://github.com/cohere-ai/cohere-python/pull/583/files#diff-1f6edbab9075bce38b9eb378de4aeb094ff60abf6e6d36b1e0d9fc11d1c029b3L167-R178

Reply:
So should  we plan on changing how mirascope interacts with Cohere ?

Reply:
I have created the PR to fix the package problem. 
https://github.com/Mirascope/mirascope/pull/620
And, I rebase the branch and change target branch of this PR.

Reply:
So lets consider this as resolved